### PR TITLE
Branch DG RosterCommand + MassFix

### PR DESCRIPTION
**Pull request by Timothyoung97**

### CODE REVIEW

**Review by michael-lee-sk — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1246,14 +1330,57 @@ testers are expected to do more *exploratory* testing.
...
+        Expected: All students whose name consists of `Alex` are shown in the student list. Lesson list will not be
+   changed. No student shown if there is no student whose name consists of `Alex`. An update message showing 
+   the number of students found will also be shown in the message box. <br>_Note: both the first and last name of the student will be checked, to see if the student's name consists of `Alex`._
+
+    * Test case: `find -`, `find .`, `find '`<br>
+      Expected: No student will be found since name should only contain alphanumeric characters and spaces.
+   
+#### Rostering
+
+1. Rostering a lesson by `LESSON_INDEX` while all students and lessons are being shown
```
Comment (eezj35) [inline docs/DeveloperGuide.md:1355]: Maybe can use another word instead of rostering? I think something like view a lesson's roster or something is better

Comment (Timothyoung97) [reply docs/DeveloperGuide.md]: Yup changed

Comment (Timothyoung97) [reply docs/DeveloperGuide.md]: Changed to View a lesson's roster

**Comment by Timothyoung97** [comment]
Check preview [here](https://timothyoung97.github.io/tp/)

**Review by eezj35 — APPROVED** [review]
LGTM

**Pull request merged**